src/formats/jpeg/utils.zig

```diff
@@ -2,6 +2,7 @@
 const std = @import("std");
 
 // See figure A.6 in T.81.
+// zig fmt: off
```

Comment:
Can you turn zig fmt on after the array?

Reply:
oops yep fixed that!